varia/sass/blocks/pullquote/_style.scss

```diff
@@ -27,44 +27,15 @@
 		color: #{map-deep-get($config-global, "color", "foreground", "light")};
 		font-size: #{map-deep-get($config-global, "font", "size", "sm")};
 		letter-spacing: #{map-deep-get($config-global, "font", "letter-spacing", "sm")};
-		display: block;
-	}
-
-	/**
-	 * Block Options
-	 */
-	&:not(.is-style-solid-color) {
-		background: none;
-
-		blockquote {
-			padding-left: 0;
-		}
 	}
 
-	&.is-style-default {
-		&.alignleft,
-		&.aligncenter,
-		&.alignright {
-			blockquote > * {
-				text-align: center;
-			}
-		}
+	blockquote {
+		padding-left: 0;
 	}
 
 	&.is-style-solid-color {
-
 		color: #{map-deep-get($config-global, "color", "background", "default")};
 
-		&:not(.alignleft):not(.alignright) blockquote {
-			@extend %responsive-aligndefault;
-		}
-
-		blockquote {
-			padding-left: #{map-deep-get($config-global, "spacing", "unit")};
-			padding-right: #{map-deep-get($config-global, "spacing", "unit")};
-			max-width: inherit;
```

Comment:
This is changing how it looks on the frontend now compared to before this PR. It is closer to how it looks in the editor but I don't know which of the two options is the correct one.

Reply:
We discussed this on slack and decided to go with this  for now.

Reply:
LGTM!